indico/modules/events/papers/templates/_contributions.html

```diff
@@ -53,14 +52,17 @@ <h2>{% trans %}Editing{% endtrans %}</h2>
                     <h3 class="icon-file">{% trans %}Paper{% endtrans %}</h3>
```

Comment:
This heading, and "Paper submission is open" / "You can submit a paper" further below, don't really apply in case of other editable types. This is especially misleading if you don't have Paper editing enabled or already submitted one and can only submit in some other editable type - its name is not mentioned anywhere until you click the button in this case.

Reply:
Example where I submitted a paper, and still have to submit slides:

![image](https://user-images.githubusercontent.com/179599/79314563-19ef5980-7f02-11ea-9df0-e9ee73e64494.png)

Reply:
Maybe I could change `Paper submission is open` to `Submission is open` and then change `You can submit a paper` to sth like `You can submit file(s) to the following types: poster,slides`

Reply:
"Submission is open" sounds good (and just get rid of the "Paper" heading I guess?)

Maybe "You can submit files for editing." for the smaller text below? With the dropdown we don't need another place listing the types, and without the dropdown I would use a specific button title that mentions for which type you will submit, i.e. "Submit <paper|slides|poster>"